src/server/env.ts

```diff
@@ -0,0 +1,24 @@
+import { config } from 'dotenv'
+import makeParser from '../lib/makeParser'
+
+config()
+
+interface Env {
+  LOG_LEVEL: string
+  LOG_PRETTY: boolean
+  NODE_ENV: 'development' | 'production' | 'test'
+  PORT: number
+}
+
+const parse = makeParser<Env>({
+  type: 'object',
+  properties: {
+    LOG_LEVEL: { type: 'string', default: 'info' },
+    LOG_PRETTY: { type: 'boolean', default: false },
+    NODE_ENV: { type: 'string', default: 'development' },
+    PORT: { type: 'number', default: 3000 },
+  },
+  required: [],
+})
+
+export default parse((process.env as unknown) as Env)
```

Comment:
**nitpick**:

ok, this might be wishful thinking... but is there any way we could somehow export this in a way we could import like:

```typescript
import { PORT } from './env'
```

this is already pretty cool, but this would be very convenient.